src/ManagedNetwork/ManagedNetwork/help/Az.ManagedNetwork.md

```diff
@@ -0,0 +1,51 @@
+---
+Module Name: Az.ManagedNetwork
+Module Guid: 00000000-0000-0000-0000-000000000000
+Help Version: 0.1.0.0
+Locale: en-US
+---
+
+# Az.ManagedNetwork Module
+## Description
+{{ Module to create, get, remove and update managednetwork }}
+
+## Az.ManagedNetwork Cmdlets
+### [Get-AzManagedNetwork](Get-AzManagedNetwork.md)
+{{ Gets a managednetwork network. }}
```

Comment:
I mean you need to run help generation tool again to get rid of these placeholders {{}}